compiler/parser/Cargo.toml

```diff
@@ -55,6 +55,11 @@ version = "1.10"
 [dependencies.tracing]
 version = "0.1"
 
+[dependencies.getrandom]
```

Comment:
Please move this dependency into alphabetical order after `[dependencies.clap]` and before `[dependencies.indexmap]`